src/service/url-replacements-impl.js

```diff
@@ -426,7 +440,7 @@ export class UrlReplacements {
    * the resulting value is `null`.
    * @param {function(!AccessService):(T|!Promise<T>)} getter
    * @param {string} expr
-   * @return {T|null}
+   * @return {Promise<?AccessService>}
```

Comment:
Hmm. This definition doesn't really make sense. Why does promise yield `?AccessService`